### PR TITLE
Fix the issues and get the latest changes

### PR DESCRIPTION
**Pull request by jws2103**
### Contribution Checklist

- [x] The code builds clean without any errors or warnings
- [x] The PR follows the [Contribution Guidelines](https://github.com/microsoft/chat-copilot/blob/main/CONTRIBUTING.md) and the [pre-submission formatting script](https://github.com/microsoft/chat-copilot/blob/main/CONTRIBUTING.md#development-scripts) raises no violations
- [] All unit tests pass, and I have added new tests where possible
- [x] I didn't break anyone :smile:

### CODE REVIEW

File: shared/Ocr/ConfigurationExtensions.cs [hunk 1]
```diff
@@ -28,7 +27,7 @@ public static class ConfigurationExtensions
 
                 if (tesseractOptions == null)
                 {
-                    throw new ConfigurationException($"Missing configuration for {ConfigOcrType}: {ocrType}");
+                    throw new ArgumentNullException($"Missing configuration for {ConfigOcrType}: {ocrType}");
```
Comment (nasreekar) [inline shared/Ocr/ConfigurationExtensions.cs:30]: is it FileNotFoundException or ArgumentNullException?

in the above file ConfigurationBuilderExtensions.cs, ConfigurationException is replaced by FileNotFoundException

File: webapi/Options/PromptsOptions.cs [hunk 2]
```diff
@@ -47,7 +52,7 @@ public class PromptsOptions
     /// Minimum relevance of a document memory to be included in the final prompt.
     /// The higher the value, the answer will be more relevant to the user intent.
     /// </summary>
-    internal float DocumentMemoryMinRelevance { get; } = 0.8F;
+    internal float DocumentMemoryMinRelevance { get; } = 0.66F;
```
Comment (nasreekar) [inline webapi/Options/PromptsOptions.cs:55]: is there a standard to set this value or just trial and error ?

Comment (jws2103) [reply webapi/Options/PromptsOptions.cs:55]: @nasreekar , I think this is the change from the main upstream https://github.com/microsoft/chat-copilot

File: shared/Ocr/ConfigurationExtensions.cs (hunk 1 above)
Comment (jws2103) [reply shared/Ocr/ConfigurationExtensions.cs:30]: @nasreekar , I think this is the change from the main upstream https://github.com/microsoft/chat-copilot

**Pull request merged**